data/brands/amenity/bicycle_repair_station.json

```
@@ -0,0 +1,32 @@
+{
+  "properties": {
+    "path": "brands/amenity/bicycle_repair_station",
+    "skipCollection": true
+  },
+  "items": [
+    {
+      "displayName": "RadKULTUR",
+      "id": "radkultur",
+      "locationSet": {"include": ["de"]},
```

We do have a geojson just for Baden-Württemberg..
```suggestion
      "locationSet": {"include": ["de-bw.geojson"]},
```